wicker/core/storage.py

```diff
@@ -461,9 +480,29 @@ class S3PathFactory(WickerPathFactory):
                     - _SUCCESS
                     - part-0-attempt-1234.parquet
                     - part-1-attempt-2345.parquet
+
+    If you supply a mount path your file paths will look like below
```

Comment:
I think you could simplify this comment and just say that if you supply a mount path it will be used to prefix all other paths (or something similar).